devel/external/AEGIS/AEGIS_25.2_training_sample_readme.txt

```diff
@@ -0,0 +1,109 @@
+This README file describes the AEGIS F814W<25.2 training sample, which is a
+randomly ordered sample of galaxies in the AEGIS field with AB magnitude<25.2, in F606W(V) and F814W(I) bands.
+
+NOTE: The columns in the catalog are those expected by galsim.RealGalaxyCatalog.
+However, some parts of that analysis, i.e parametric fits, were not performed.
+Hence those values were manually set and do not have physical meaning.
+
+(1) The catalogs themselves are in AEGIS_galaxy_catalog_[VI]_25.2.fits (where
+[VI] indicates either V or I).  These files can be used to initialize a
+galsim.RealGalaxyCatalog or galsim.COSMOSCatalog instance, and contain a FITS
+tables that could also be  read in directly using, e.g., python or IDL.  The
+fields that are included for each galaxy are:
+
+IDENT: AEGIS object identifier. This value is internally derived.  Therefore
+do not rely on this IDENT for cross-matching between other AEGIS catalogs.
+
+RA, DEC: Right ascension and declination (J2000, degrees).  While this is not
+needed for image simulation, it is necessary for proper cross-matching with
+other catalogs.
+
+MAG: magnitude (MAG_AUTO calculated from sextractor).
+
+BAND: Passband for the observed image (F814W/F606W).
+
+WEIGHT: A weight factor to use to account for the fact that larger galaxies are
+preferentially excluded from this catalog due to the proximity to CCD edges.
+(Currently all weights are set to 1.0).
```

Comment:
I reworked some portion of the code and included some selection cuts. 26,517 galaxies  pass the cuts out of the total 28,532 galaxies with I band magnitude <25.2. Previously with `exclusion_level=bad_stamp` I was losing about 10% galaxies. I visually inspected the galaxies and realized that most of these were large galaxies for which HSM adaptive moments failed to converge. I altered the input HSM params and most of these passed, except galaxies with high ellipticities. So I decided to include galaxies with `ELLIPTICITY` >0.75 computed by SExtractor, even if  adaptive moments were not computed; I set `sn_ellip_gauss`= -10 manually.

A galaxy is included in the final catalog if:
1)  `sn_ellip_gauss` > 0 or ELLIPTICITY >0.75, in both filters, and 
2) Closest masked pixel is more than 5 pixels from the center, and
3) If closest masked pixel is closer than 11 pixels, `average_mask_adjacent_pixel_count` must be less than 20% of the brightest pixel.

With these cuts, this is the distribution of galaxies that would make it to the final catalog. The left column is the fraction of galaxies that pass selection cuts as a function of half light radius, ellipticity and position angle (measured by SExtractor), and magnitude. The right column is the histogram of all galaxies(without applying cuts)  as a function of the same parameters.

![screen shot 2016-11-01 at 2 19 49 pm](https://cloud.githubusercontent.com/assets/13785537/19907953/5001a2d6-a03e-11e6-8f6f-8b82017d0ce0.png)

The size dependent selection effect I get is different from COSMOS. Out of the 32 galxies larger than 1.4 arcsec only 6 fail the selection cut.  At this point should any other selection cuts be applied? 

From the COSMOS <25.2 catalog, I plotted the weight and  flux radius for the galaxies. The histogram of the the flux radius seems to resemble the distribution for AEGIS. 

![screen shot 2016-11-01 at 2 40 17 pm](https://cloud.githubusercontent.com/assets/13785537/19908565/24a33ca0-a041-11e6-8b71-0e05d2506fe3.png)

@rmandelb  I'm not clear as to how the weight was computed for COSMOS?